code/modules/power/singularity/narsie.dm

```diff
@@ -39,6 +41,29 @@
 /obj/narsie/Initialize(mapload)
 	. = ..()
 
+	narsie_spawn_animation()
+
+/obj/narsie/Destroy()
+	if (GLOB.cult_narsie == src)
+		fall_of_the_harbinger()
+		GLOB.cult_narsie = null
+
+	return ..()
+
+/// This proc sets up all of Nar'Sie's abilities, stats, and begins her round-ending capabilities. She does not do anything unless this proc is invoked.
+/// This is only meant to be invoked after this instance is initialized in specific pro-sumer procs, as it WILL derail the entire round.
+/// Pass in the user that invoked this proc ONLY if it was called via View Variables, if this is invoked through normal gameplay please do pass in null.
+/obj/narsie/proc/start_ending_the_round()
+	var/mob/potential_user = null
+	if(IsAdminAdvancedProcCall())
```

Comment:
Do we really need this? This makes it impossible for something like Lua scripts to create Nar'Sie. The proc name is extremely obvious.